scripts/generate_llava_ov2_thumbnail.py

```
@@ -0,0 +1,137 @@
+from PIL import Image, ImageEnhance, ImageFilter, ImageOps, ImageDraw
+
+# ---------- Brand palette (matches onevision_encoder.png) ----------
+BG_COLOR = (10, 12, 18)        # near-black
+DOT_COLOR = (240, 230, 208)    # cream
+
+W, H = 1456, 816
+DOT = 4
+R_MAX = DOT * 0.50
+
+ROOT = Path(__file__).resolve().parent.parent
+SOURCE_PATH = Path("/tmp/source_image.jpg")
```

Ruff (0.15.15)

[error] Probable insecure usage of temporary file or directory: "/tmp/source_image.jpg"

(S108)